app/views/dashboard/firms/index.html.erb

```diff
@@ -19,25 +19,28 @@
     </table>
   </div>
 
-  <h2><%= t('dashboard.firms_index.trading_names_heading') %></h2>
-  <div class="table-wrapper">
-    <table>
-      <thead>
-        <tr>
-          <th><%= Firm.human_attribute_name(:fca_number) %></th>
-          <th><%= Firm.human_attribute_name(:registered_name)  %></th>
-          <th><%= t('dashboard.firms_index.principal_column_heading') %></th>
-        </tr>
-      </thead>
-      <tbody>
-        <% @trading_names.each do |trading_name| %>
-          <tr class="t-trading-name-table-row">
-            <td class="t-frn"><%= trading_name.fca_number %></td>
-            <td class="t-firm-name"><%= trading_name.registered_name %></td>
-            <td class="t-principal-name"><%= trading_name.principal.full_name %></td>
+  <div id="firm-list">
+    <input class="fuzzy-search firm-list__filter" placeholder="filter firms"/>
```

Comment:
As discussed on Slack with @neoeno, we can't use `placeholder` to act as a label, it should be for example input text only and considered non-essential.